dat/events/proteron/blockade.lua

```diff
@@ -36,14 +36,15 @@ local function spawn_fleet( pos )
    end
 
    local plts = {}
+   local stand, _str = faction.get("Proteron"):playerStanding()
    for k,s in ipairs(ships) do
       local leader = plts[1]
       local p = pilot.add( s, "Proteron", pos, nil, {ai="guard"} )
       local aimem = p:memory()
       aimem.enemyclose    = 10e3
       aimem.guarddodist   = 10e3
       aimem.guardreturndist = 15e3
-      p:setHostile(true) -- TODO change this when something can make them non-hostile
+      p:setHostile(stand<0)
```

Comment:
Strictly speaking, I don't think this is necessary, because this is the default behaviour.

Reply:
Ah, I see, fair enough